src/ucp/rndv/rndv.c

```diff
@@ -426,146 +424,138 @@ static void ucp_rndv_req_send_rtr(ucp_request_t *rndv_req, ucp_request_t *rreq,
     ucp_request_send(rndv_req, 0);
 }
 
-static ucp_lane_index_t
-ucp_rndv_get_zcopy_get_lane(ucp_request_t *rndv_req, uct_rkey_t *uct_rkey)
+static ucp_lane_index_t ucp_rndv_zcopy_get_lane(ucp_request_t *rndv_req,
+                                                uct_rkey_t *uct_rkey,
+                                                int is_get_zcopy)
```

Comment:
can use here `proto` instead of `is_get_zcopy` flag for consistency with other funcs